api/agent/drivers/docker/docker_test.go

```diff
@@ -537,3 +540,33 @@ func TestRunnerDockerNoNetwork(t *testing.T) {
 	// this is unnecessary as NetworkMode=none is well known. (eg. docker run --network none)
 	// https://docs.docker.com/network/none/
 }
+
+func TestRunnerDockerInvalidSyslog(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(30)*time.Second)
+	defer cancel()
+
+	dkr := NewDocker(drivers.Config{})
+	defer dkr.Close()
+
+	task := createTask("test-docker-no-net")
+	task.logURL = "tcp://invalid:9999"
```

Comment:
I think we can use: 192.0.2.0 (https://tools.ietf.org/html/rfc5737 TEST-NET1) for this.. Minor nit...